### DISCUSSION TITLE
Specify Start and End time

### DISCUSSION THREAD

**Discussion by MickeyM007** [MacielG1/VodRecovery#8] [Q&A]
I'm back haha. 
I just find out in v1.2.7, when I specify time (in 1-VOD Recovery or 3-Download VOD (.mp4)) I get an error: 
```
An error occurred: [WinError 2] The system cannot find the file specified
Press Enter to exit.
```
Is there anything I can do?
Normal download works fine.

**Comment by MacielG1** [comment]
This update should address the issue: https://github.com/MacielG1/VodRecovery/releases/tag/1.2.8

**Reply by MickeyM007** [reply]
Yes, fixed! Thank you! I see ffprobe is now necessary.
